app/data/models/claim.js

```diff
@@ -35,7 +38,7 @@ module.exports = (sequelize, DataTypes) => {
       tableName: 'claim',
       hooks: {
         afterCreate: async (claimRecord, _) => {
-          claimRecord.dataValues.reference = createReference(claimRecord.id)
+          claimRecord.dataValues.reference = endemics.enabled ? createAgreementNumber(generateClaimPreText(claimRecord.type, claimRecord.dataValues.data.typeOfLivestock)) : createReference(claimRecord.id)
```

Comment:
Why we have to check Endemics Flag, this table won't have any value if its not endemics.

Reply:
thats what Alex said to do since its a new world solution